src/cairoUtilFuncGen/storage/dynArrayPushWithoutArg.ts

```diff
@@ -34,7 +27,7 @@ export class DynArrayPushWithoutArgGen extends StringIndexedFuncGen {
       [['newElemLoc', typeNameFromTypeNode(elementType, this.ast), DataLocation.Storage]],
       ['syscall_ptr', 'pedersen_ptr', 'range_check_ptr'],
       this.ast,
-      nodeInSourceUnit ?? push,
+      this.sourceUnit,
```

Comment:
Is there a reason for doing this rather than passing in a valid nodeInSourceUnit parameter?